packages/app/src/utils/API/nominationsAPI.js

```diff
@@ -2,12 +2,19 @@ import axios from 'axios';
 
 const API_URL = process.env.REACT_APP_API_URL
 
+const cleanURL = (str) => {
+  if (str.endsWith('/')) {
+    return str.slice(0, -1);
+  }
+  else return str;
```

Comment:
one small nit pick about this 

I would recommend wrapping the `return str` in `{ }` for consistency with the if statement above. 

OR

Because javascript will stop processing the block of code when a `return` statement is processed you can remove the `else` from the `else return str`.